playgrounds/react/examples/popup-example.tsx

```diff
@@ -21,7 +21,7 @@ export default class extends React.Component<any, { visible: boolean; text: stri
     this.handleTextUpdate = this.handleTextUpdate.bind(this);
   }
 
-  private handleTextUpdate(e: unknown) {
+  private handleTextUpdate(e: any) {
```

Comment:
Don't we have a better option to avoid `any`?

Reply:
The examples in the playground are ancient (pre-TS improvements). I proposed to remove them to simplify the playground, but the team suggested to leave it be for now just in case. Anyway, the examples need a reboot in my opinion, but I removed all 'any' types for now.